### PR TITLE
Add a new method to get related places.

### PR DESCRIPTION
**Pull request by Spaceenter**

### CODE REVIEW

File: datacommons/places.py [hunk 1]
```diff
@@ -26,47 +26,113 @@
...
+        "gender": "Female"
+      }, "count", "measured")
+      {
+        'geoId/06085': [
+          'geoId/06041',
+          'geoId/06089',
+          'geoId/06015',
+          'geoId/06023',
+        ]
+      }
```
Comment (pradh) [inline datacommons/places.py]: Need to update example?

File: datacommons/places.py [hunk 2]
```diff
@@ -26,47 +26,113 @@
...
+
+
+def get_related_places(dcids, population_type, constraining_properties={},
+                       measured_prop='count',
+                       stat_type='measured', within_place='',
+                       per_capita=False, same_place_type=False):
+    """ Returns :obj:`Place`s related to :code:`dcids` for the given constraints.
+
+    Args:
+      dcids (:obj:`iterable` of :obj:`str`): Dcids to get contained in places.
```
Comment (shifucun) [inline datacommons/places.py]: related

File: datacommons/places.py [hunk 3]
```diff
@@ -26,47 +26,113 @@
...
+        'place_type': place_type,
+    })
+
+    # Create the results and format it appropriately
+    result = utils._format_expand_payload(payload, 'place', must_exist=dcids)
+    return result
+
+
+def get_related_places(dcids, population_type, constraining_properties={},
+                       measured_prop='count',
```
Comment (shifucun) [inline datacommons/places.py]: measured_property

File: datacommons/places.py [hunk 4]
```diff
@@ -26,47 +26,113 @@
...
+
+    Args:
+      dcids (:obj:`iterable` of :obj:`str`): Dcids to get contained in places.
+      population_type (:obj:`str`): The type of statistical population.
+      constraining_properties (:obj:`map` from :obj:`str` to :obj:`str`, optional):
+        A map from constraining property to the value that the
+        :obj:`StatisticalPopulation` should be constrained by.
+      measured_property (:obj:`str`): The measured property.
+      stat_type (:obj:`str`): The statistical type for the observation.
+      within_place(:obj:`str`): Optional, the place that all the related places
```
Comment (shifucun) [inline datacommons/places.py]: place dcid?

File: datacommons/places.py [hunk 5]
```diff
@@ -26,47 +26,113 @@
...
+        are contained in.
+      per_capita(:obj:`bool`): Optional, whether to take into account
+        `PerCapita` when compute the relatedness.
+      same_place_type(:obj:`bool`): Optional, whether to require all the
+        related places under the same ancestor place.
+
```
Comment (shifucun) [inline datacommons/places.py]: same place type?

File: datacommons/places.py [hunk 6]
```diff
@@ -26,47 +26,113 @@
...
+      ValueError: If the payload returned by the Data Commons REST API is
+        malformed.
+
+    Examples:
+      We would like to get all related places of
+      `Santa Clara county <https://browser.datacommons.org/kg?dcid=geoId/06085>`
+      Specifying the :code:`dcids` as a :obj:`list` result in the following.
+
+      >>> get_places_in(["geoId/06"], "Person", {
+        "age": "Years21To64",
```
Comment (shifucun) [inline datacommons/places.py]: get_related_places

File: datacommons/places.py (hunk 1 above)
Comment (Spaceenter) [reply datacommons/places.py]: The examples just show the format -- the result would change as we change algorithms to compute relatedness.

**Pull request merged**